.codeclimate.yml

```diff
@@ -1,6 +1,8 @@
 ---
 version: 2
 plugins:
+  duplication:
```

Comment:
Instead of re-enabling this, you'll want to disable these checks for version 2.
```
checks:
  similar-code:
    enabled: false
  identical-code:
    enabled: false
```

https://docs.codeclimate.com/docs/duplication-concept